docs/hugo/content/ADR-2023-03-Post-Reconciliation-Extension.md

```diff
@@ -0,0 +1,46 @@
+---
+title: '2023-03: Post Reconciliation Extensions'
+---
+
+## Context
+
+The current behaviour of Azure Service Operator (ASO) is to set the `Ready` status on a resource by default after a successful creation of a resource. 
+
+For some resources, this is not ideal as we need to check the status of a created resource post reconciliation and set a condition accordingly. 
+
+An example of this:
+
+- `PrivateEndpoints` with `ConnectionState` of `Approved`, `Rejected` or `Pending`.
```

Comment:
If there's one example, put it in  sentence. Only use a bullet point if there's more than one.

Reply:
Done!